src/it/scala/ch/epfl/bluebrain/nexus/perf/CreateSimulationNoSchema.scala

```diff
@@ -0,0 +1,58 @@
+package ch.epfl.bluebrain.nexus.perf
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.perf.data.generation.types.Settings
+import ch.epfl.bluebrain.nexus.perf.data.generation.{ResourcesGenerator, Templates}
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class CreateSimulationNoSchema extends BaseSimulation {
+
+  private val projectNumber = config.createConfig.project.toString
+  private val size          = config.createConfig.size
+  private val parallelUsers = config.createConfig.parallelUsers
+
+  private val map = Map[String, Uri](
```

Comment:
why do we need this map then?

Reply:
This is used by the `ResourceGenerator` I believe.